plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -465,7 +465,7 @@ public FSDataInputStream open(Path path, int bufferSize)
     public FSDataOutputStream create(Path path, FsPermission permission, boolean overwrite, int bufferSize, short replication, long blockSize, Progressable progress)
             throws IOException
     {
-        // Ignore the overwrite flag, since Trino always writes to unique file names.
+        // Ignore the overwrite flag, since Trino Hive connector *usually* writes to unique file names.
```

Comment:
As a naive reader of this comment, I'm a bit puzzled and ask myself what are the "unusual" use cases when the file names are not unique (since "usually" is brought in the sentence).

Reply:
This is change "always -> usually".  I can choose a different wording, but i want to convey

- hive connector typically uses unique files names 
- it not always does so

this is not the place to elaborate more, or go into details.